news/2023/2023-02-17-community-contributors-elite-nominators-2022.md

```diff
@@ -0,0 +1,125 @@
+
+[![](/wiki/shared/news/2023-02-17-community-contributors-elite-nominators-2022/fieryrage.jpg)](https://osu.ppy.sh/users/3533958)
+
+A long-time member of the Beatmap Nominators, [fieryrage](https://osu.ppy.sh/users/3533958), really popped off this year, well and truly leaving his mark throughout the ranked section.
+
+His overall performance was excellent, showcasing a high degree of activity while still maintaining consistent attention to detail and thorough modding — certainly no small feat for anyone familiar with the task! A mapper could rarely say “I was [By Myself](https://osu.ppy.sh/beatmapsets/1152507)” in 2022 with how often fieryrage helped both new and returning ones alike rank their maps!
+
+## osu!taiko
+
+[![](/wiki/shared/news/2023-02-17-community-contributors-elite-nominators-2022/Hivie.jpg)](https://osu.ppy.sh/users/14102976)
+
+*Note: we are obligated to inform readers that [radar](https://osu.ppy.sh/users/7131099) is responsible for writing this segment.*
+
+[Hivie](https://osu.ppy.sh/users/14102976) is back again to prove that anything is possible with enough shawarma and beautiful anime females. Rising to the top once again and contributing immensely to everything inside and out of the Beatmap Nominators, managing to reach over 200 nominations with barely any issues is just the first thing on his growing list of accomplishments. 
+
+He has also run some of the most successful mapping contests the mode has ever seen in the form of [TFAC](/wiki/Contests/o%21tFAC) and [M2P](/wiki/Contests/M2P), and created a [Discord bot](https://osu.ppy.sh/community/forums/topics/1604925) for mapping and modding related statistics which has seen prevalence in over 380 Discord servers! 
+
+Not to mention that this was all done while consistently maintaining his leadership in Project Loved, Beatmap Spotlights, and the Community Mentorship Program. We hope to keep him kidnapped at the NAT headquarters for as long as possible, as who knows what could go wrong if we set him loose upon the world without restraint. 
+
+[![](/wiki/shared/news/2023-02-17-community-contributors-elite-nominators-2022/meiqth.jpg)](https://osu.ppy.sh/users/12565402)
+
+Joining the Beatmap Nominators in June 2021, [meiqth](https://osu.ppy.sh/users/12565402) has been consistently performing at a top level with zero signs of slowing down, and to the surprise of no one, they took 2022 by storm with a consistently excellent performance in nominating a very diverse catalogue of over 200 beatmaps, almost entirely without issues. 
+
+Not only satisfied with merely that, they also went out of their way to be one of the very few people who actively tackled the daunting task of Quality Assurance in order to make sure no sneaky unrankables or quality issues accidentally make their way into the Ranked section, nipping problems in the bud long before they ever get to inconvenience players.
```

Comment:
```suggestion
Not only satisfied with merely that, they also went out of their way to be one of the *very* few people who actively tackled the daunting task of Quality Assurance in order to make sure no sneaky unrankables or quality issues accidentally make their way into the Ranked section, nipping problems in the bud long before they ever get to inconvenience players.
```

would like to emphasize "very" cuz barely anyone else did qah with a decent pace in general